README.md

```diff
@@ -1,104 +1,131 @@
 ```
 
+
 ## How to use
 
-#### Loading the editor
+### Loading the editor
+
+After installing and/or importing the module, you can load EasyMDE onto the first TextArea on the web page:
 
-After installing and/or importing the module, you can load EasyMDE onto the first TextArea on the webpage.
 ```html
 <textarea></textarea>
 <script>
 var easyMDE = new EasyMDE();
 </script>
 ```
 
-Alternatively you can select a specific TextArea, via Javascript.
+Alternatively you can select a specific TextArea, via Javascript:
+
 ```html
 <textarea id="my-text-area"></textarea>
 <script>
 var easyMDE = new EasyMDE({element: document.getElementById('my-text-area')});
 </script>
 ```
 
-Or via jQuery.
+Or via jQuery:
+
 ```html
 <textarea id="my-text-area"></textarea>
 <script>
 var easyMDE = new EasyMDE({element: $('#my-text-area')[0]});
 </script>
 ```
 
-#### Editor functions
 
-Use EasyMDE.value() to get the content of the editor.
+### Editor functions
+
+Use EasyMDE.value() to get the content of the editor:
+
 ```html
 <script>
 easyMDE.value();
 </script>
 ```
 
-Use EasyMDE.value(val) to set the content of the editor.
+Use EasyMDE.value(val) to set the content of the editor:
+
 ```html
 <script>
 easyMDE.value('New input for **EasyMDE**');
 </script>
 ```
 
+
 ## Configuration
 
+### Options list
+
 - **autoDownloadFontAwesome**: If set to `true`, force downloads Font Awesome (used for icons). If set to `false`, prevents downloading. Defaults to `undefined`, which will intelligently check whether Font Awesome has already been included, then download accordingly.
-- **autofocus**: If set to `true`, autofocuses the editor. Defaults to `false`.
+- **autofocus**: If set to `true`, focuses the editor automatically. Defaults to `false`.
 - **autosave**: *Saves the text that's being written and will load it back in the future. It will forget the text when the form it's contained in is submitted.*
-  - **enabled**: If set to `true`, autosave the text. Defaults to `false`.
+  - **enabled**: If set to `true`, saves the text automatically. Defaults to `false`.
   - **delay**: Delay between saves, in milliseconds. Defaults to `10000` (10s).
   - **uniqueId**: You must set a unique string identifier so that EasyMDE can autosave. Something that separates this from other instances of EasyMDE elsewhere on your website.
 - **blockStyles**: Customize how certain buttons that style blocks of text behave.
   - **bold**: Can be set to `**` or `__`. Defaults to `**`.
   - **code**: Can be set to  ```` ``` ```` or `~~~`.  Defaults to ```` ``` ````.
   - **italic**: Can be set to `*` or `_`. Defaults to `*`.
-- **element**: The DOM element for the textarea to use. Defaults to the first textarea on the page.
-- **forceSync**: If set to `true`, force text changes made in EasyMDE to be immediately stored in original textarea. Defaults to `false`.
+- **element**: The DOM element for the text area to use. Defaults to the first text area on the page.
```

Comment:
"textarea" refers to the HTML DOM element `<textarea>`, that is written as one word.